essential-documentation/contribute-to-appflowy/blog-post-work-in-progress/appflowy-with-supabase-wip.md

```diff
@@ -1,40 +1,54 @@
-# AppFlowy with Supabase
+# Authentication and Data Storage for AppFlowy Using Supabase
 
-## AppFlowy with Supabase
+## Introducing Cloud Enabled AppFlowy
 
-AppFlowy is built with user-friendly local app features in mind. Having noted the frequent requests for a cloud version, we're pleased to share that AppFlowy can now be cloud-enabled. We've partnered with Supabase, an open-source alternative to Firebase, to handle user sign-ins and data storage. With a Supabase project, transforming AppFlowy into a self-hosted cloud app is straightforward. By the end of this article, you'll be primed to launch your very own AppFlowy cloud application with ease. It's worth noting that the steps for other cloud services, such as `Appwrite` or `Firebase`, bear similarities."
+AppFlowy is built with local app features in mind to keep it simple and user friendly. However, given the frequent requests for a cloud version, we're pleased to share that AppFlowy can now be cloud-enabled. 
+
+In order to implement cloud computimng features such as user sign-ins and data storage, we've partnered with [Supabase](https://supabase.com/), an open-source alternative to [Firebase](https://firebase.google.com/).  
```

Comment:
computimng => computing?

Reply:
fixed (just removed the word -- I'm not sure it really adds meaning)